### PR TITLE
Create guide_phpservermonitor.rst

### PR DESCRIPTION
**Pull request by thomas-kammerer**

### CODE REVIEW

File: source/guide_phpservermonitor.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Installation
+============
+
+To install PHP Server Monitor we download the current version from the github release. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the zip file will be under your ``html``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/phpservermon/phpservermon/releases/download/v3.4.5/phpservermon-3.4.5.zip
```
Comment (luto) [inline source/guide_phpservermonitor.rst]: you can use `https://github.com/phpservermon/phpservermon/archive/v3.4.5.tar.gz` here. the URL is shorter and it enables the next step.

File: source/guide_phpservermonitor.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+ [isabell@stardust html]$ unzip phpservermon-3.4.5.zip
+ [isabell@stardust html]$ rm phpservermon-3.4.5.zip
+ [isabell@stardust html]$ mv phpservermon-3.4.5/* phpservermon-3.4.5/.* ./
```
Comment (luto) [inline source/guide_phpservermonitor.rst]: use `tar --strip-components 1` to remove the top-level directory.

File: source/guide_phpservermonitor.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Fill out your system settings, admin user and edit the following database settings:
+ * PSM_DB_USER: ``isabell``
+ * PSM_DB_PASS from your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>`
+ * PSM_DB_NAME: ``isabell_psm``
+
+::
+
+Finally, configure a cronjob: Add the following line to your crontab using the crontab -e command:
+
+* * * * * /usr/bin/php /var/www/virtual/isabell/html/cron/status.cron.php
```
Comment (luto) [inline source/guide_phpservermonitor.rst]: The `::` from line 79 has to move down a bit [to fix the formatting](https://deploy-preview-571--uberlab.netlify.com/guide_phpservermonitor.html#configuration).

Comment (luto) [inline source/guide_phpservermonitor.rst]: How about `$USER` here to make the line copy-paste-able as-is?

File: source/guide_phpservermonitor.rst (hunk 1 above)
Comment (luto) [reply source/guide_phpservermonitor.rst]: Oh, I see. The tar doesn't contain the deps, the zip does :roll_eyes:

File: source/guide_phpservermonitor.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/phpservermonitor.png
```
Comment (luto) [inline source/guide_phpservermonitor.rst]: this file is missing

**Comment by luto** [comment]
Works great :) I left a few comments. Thank you!

File: source/guide_phpservermonitor.rst (hunk 4 above)
Comment (thomas-kammerer) [reply source/guide_phpservermonitor.rst]: How can I add the image to my pull request?

"Uploads are disabled.
File uploads require push access to this repository."

Comment (luto) [reply source/guide_phpservermonitor.rst]: Did you try in this repo (uberspace/lab)? It should work inside your own branch: https://github.com/thomas-kammerer/lab/tree/patch-8/source/_static/images

**Comment by luto** [comment]
Merged, thank you! :tada:

**Pull request closed without merging**